test/elixir/test/users_db_security_test.exs

```diff
@@ -336,6 +336,14 @@ defmodule UsersDbSecurityTest do
       "3"
     })
 
+    # make changing PRF regenerates derived_key and salt on next
```

Comment:
salt is _not_ regenerated (to avoid conflicts)

Reply:
it's a copy-paste thing, I copied the setting code but forgot to update the comment